RELEASES.md

```diff
@@ -140,6 +140,28 @@ Compatibility Notes
 [`{Any + Send + Sync}::downcast_ref`]: https://doc.rust-lang.org/std/any/trait.Any.html#method.downcast_ref-2
 [`{Any + Send + Sync}::is`]: https://doc.rust-lang.org/std/any/trait.Any.html#method.is-2
 
+Version 1.27.1 (2018-07-10)
+===========================
+
+Security Notes
+--------------
+
+- rustdoc would execute plugins in the /tmp/rustdoc/plugins directory
+  when running, which enabled executing code as some other user on a
+  given machine. This release fixes that vulnerability; you can read
+  more about this on the [blog][rustdoc-sec].
```

Comment:
Can you add the CVE now that we have a number?

Reply:
I'd prefer not to; stable doesn't have the CVE and it feels odd to make those edits to "past" release notes.

Reply:
Well, we did not have the number when we released 1.27.1, and having it in there would be helpful if someone is searching when the vulnerability was fixed.

Also, we already merged other PRs changing past releases' notes (such as https://github.com/rust-lang/rust/pull/51997 and https://github.com/rust-lang/rust/pull/51868), and if we do 1.27.2 we can even get this to stable faster.